### PR TITLE
Amol | Added Transfer Out Section

### PR DESCRIPTION
**Pull request by amolbch**

### CODE REVIEW

**Comment by nthfloor** [comment]
@amolbch see the linting errors below:
Screenshot from 2019-06-14 14-33-02

File: ui/app/i18n/registration/locale_pt_BR.json [hunk 1]
```diff
@@ -152,6 +152,11 @@
     "REGISTRATION_OPTION_NONE": "Nenhum",
     "REGISTRATION_OPTION_NONE_TEXT": "Documento",
     "BMI STATUS": "Estado Nutricional",
-    "BMI": "IMC"
+    "BMI": "IMC",
+    "REGISTRATION_TRANSFER_OUT_INFO_KEY": "Transferido Para",
+    "REGISTRATION_TRANSFER_OUT_HF_NAME_INFO":"Unidade Sanitária de Destino",
+    "Is_Transferred_Out":"Data de Transferência",
```
Comment (nthfloor) [inline ui/app/i18n/registration/locale_pt_BR.json]: Should be: `Is Transferred Out`

File: ui/app/registration/views/transferOutAddressFields.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<article class="form-field" ng-class="::{'left-form-field': $index % 2 != 0 }"
+         ng-repeat="addressLevel in ::addressLevels"
+         ng-if="addressLevel.name === 'HF_NAME' || addressLevel.name === 'HF_DISTRICT' || addressLevel.name === 'HF_PROVINCE'" addressLevel="2">
+    <div class="field-attribute" ng-if="addressLevel.name!='HF_NAME'"><label for="{{::addressLevel.addressField}}">{{::addressLevel.name | translate}}</label>
+    </div>
+    <div class="field-attribute" ng-if="addressLevel.name =='HF_NAME'"><label for="{{::addressLevel.addressField}}">{{::'REGISTRATION_TRANSFER_OUT_HF_NAME_INFO' | translate}}<span class="asterick" ng-show="isRequired" >*</span></label>
```
Comment (nthfloor) [inline ui/app/registration/views/transferOutAddressFields.html:6]: This fields does not appear when the checkbox is selected.
Screenshot from 2019-06-18 09-15-54

**Review by nthfloor — CHANGES_REQUESTED** [review]
@amolbch a couple of comments. Please take a look.

File: ui/app/i18n/registration/locale_pt_BR.json (hunk 1 above)
Comment (amolbch) [reply ui/app/i18n/registration/locale_pt_BR.json]: Updated translation "Is_Transferred_Out":"Transferir Para",

**Review by nthfloor — APPROVED** [review]

**Pull request merged**